scripts/jenkins/cloud/ansible/roles/cloud_generator/templates/barclamps/octavia.yml.j2

```diff
@@ -1,2 +1,23 @@
   - barclamp: octavia
+    attributes:
+      api:
+        # HTTPs not yet supported by Octavia
+        protocol: http # {{ api_protocol }}
+{% include 'barclamps/lib/ssl.yml.j2' %}
+      certs:
+        # same passphrase used in qa_crowbarsetup.sh setup_octavia_cert to generate the keys
+        passphrase: crowbar
```

Comment:
is passphrase required?

Reply:
Yes, the Octavia barclamp requires a passphrase to be set for the server CA key. Unless we change the implementation to allow for an empty password value, this is required.

Reply:
I see. It is optional in upstream code. But that's fine. We don't have to change our barclamp at this point, unless customer finds it annoying.